### ISSUE TITLE
Where is the test/evaluation code?

### ISSUE THREAD

**Issue by happynear** [yifan123/flow_grpo#13]
I have trained an OCR reward model using your code and the reward curve looks good. Now I want to evaluate it but find no script in this repo. Could you provide an evaluation code?

**Comment by happynear** [comment]
Thanks for your reply!
I can find some examples on wandb, and they look much better than the base model. But I also want to get a numerical result on the test set. Do you have such code?

**Comment by yifan123** [comment]
You can check `eval_reward_ocr` in wandb, which corresponds to the accuracy on the evaluation/validation set. Our reward definition is essentially aligned with accuracy.

**Comment by happynear** [comment]
Oh I find it. Thanks for your quick reply again!
